roles/addons/prometheus_server/tasks/main.yml

```diff
@@ -30,6 +30,35 @@
     - monitoring.exporters.snmp_exporter.with_generator is defined      
     - monitoring.exporters.snmp_exporter.with_generator
 
+- name: "copy █ Install firewalld service configuration file"
+  copy:
+    src: alertmanager.xml
+    dest: /usr/lib/firewalld/services/alertmanager.xml
```

Comment:
Please prefer /etc/firewalld/services/alertmanager.xml. (https://firewalld.org/documentation/configuration/directories.html)

Reply:
I took your snippet, didnt knew about /etc/firewalld, thanks.

Reply:
You did fast! I updated the snippet one minute after posting when I saw my mistake 😄